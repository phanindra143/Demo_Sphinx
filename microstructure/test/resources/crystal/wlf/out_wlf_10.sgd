*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=    84.6441
  !k_ij(1,2)=     0.7258
  !k_ij(2,2)=    84.3092
  !k_ij(1,3)=     0.1517
  !k_ij(2,3)=     2.0795
  !k_ij(3,3)=    84.3685
MP,Kxx,1,   84.64412
MP,Kyy,1,   84.30922
MP,Kzz,1,   84.36851
MP,Kxx,2,   84.30922
MP,Kyy,2,   84.64412
MP,Kzz,2,   84.36851
MP,Kxx,3,   84.30922
MP,Kyy,3,   84.64412
MP,Kzz,3,   84.36851
MP,Kxx,4,   84.64412
MP,Kyy,4,   84.30922
MP,Kzz,4,   84.36851
MP,Kxx,5,   84.64412
MP,Kyy,5,   84.30922
MP,Kzz,5,   84.36851
MP,Kxx,6,   84.30922
MP,Kyy,6,   84.64412
MP,Kzz,6,   84.36851
MP,Kxx,7,   84.30922
MP,Kyy,7,   84.64412
MP,Kzz,7,   84.36851
MP,Kxx,8,   84.64412
MP,Kyy,8,   84.30922
MP,Kzz,8,   84.36851
MP,Kxx,9,   84.36851
MP,Kyy,9,   84.64412
MP,Kzz,9,   84.30922
MP,Kxx,10,   84.64412
MP,Kyy,10,   84.36851
MP,Kzz,10,   84.30922
MP,Kxx,11,   84.64412
MP,Kyy,11,   84.36851
MP,Kzz,11,   84.30922
MP,Kxx,12,   84.36851
MP,Kyy,12,   84.64412
MP,Kzz,12,   84.30922
MP,Kxx,13,   84.36851
MP,Kyy,13,   84.64412
MP,Kzz,13,   84.30922
MP,Kxx,14,   84.64412
MP,Kyy,14,   84.36851
MP,Kzz,14,   84.30922
MP,Kxx,15,   84.64412
MP,Kyy,15,   84.36851
MP,Kzz,15,   84.30922
MP,Kxx,16,   84.36851
MP,Kyy,16,   84.64412
MP,Kzz,16,   84.30922
MP,Kxx,17,   84.30922
MP,Kyy,17,   84.36851
MP,Kzz,17,   84.64412
MP,Kxx,18,   84.36851
MP,Kyy,18,   84.30922
MP,Kzz,18,   84.64412
MP,Kxx,19,   84.36851
MP,Kyy,19,   84.30922
MP,Kzz,19,   84.64412
MP,Kxx,20,   84.30922
MP,Kyy,20,   84.36851
MP,Kzz,20,   84.64412
MP,Kxx,21,   84.30922
MP,Kyy,21,   84.36851
MP,Kzz,21,   84.64412
MP,Kxx,22,   84.36851
MP,Kyy,22,   84.30922
MP,Kzz,22,   84.64412
MP,Kxx,23,   84.36851
MP,Kyy,23,   84.30922
MP,Kzz,23,   84.64412
MP,Kxx,24,   84.30922
MP,Kyy,24,   84.36851
MP,Kzz,24,   84.64412
*endif
