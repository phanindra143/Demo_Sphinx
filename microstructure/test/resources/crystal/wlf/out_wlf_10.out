            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\15
             3.00000      -1322732.18630        -31577.86504         23320.36694
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           182.16343          -336.17367
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000        -30931.08705      -1316068.62767         13958.92465
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           299.59499         15625.00000          -630.60208
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         11516.61519         -5058.51870      -1318046.98035
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000          -130.70319          -245.79083         15625.00000
  /%
